pyinstaller.py

```diff
@@ -119,20 +128,67 @@
     cipher=block_cipher,
     noarchive=False,
 )
-pyz = PYZ(a.pure, a.zipped_data, cipher=block_cipher)
-exe = EXE(
-    pyz,
+a_pyz = PYZ(a.pure, a.zipped_data, cipher=block_cipher)
```

Comment:
The idea here being to remove/refactor this when we get containerized canu rolling?

Reply:
Yes.  This goes away completely in the follow up containerization PR